listenbrainz_spark/stats/user/tests/test_utils.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
remove the lines. If these lines are automatically added by your text-editor, please change the settings.